extras/cloudshell/tutorial.md

```diff
@@ -100,15 +105,16 @@ To stop running the app:
 
 2. Click the 'Stop' icon. 🟥
 
-3. Select **Yes** to clean up deployed resources. 
+3. If prompted, select **Yes** to clean up deployed resources.
 
 You can start, stop, and debug apps from the Debug view.
 
+### Cleanup
 If you've deployed your app to a GKE cluster in your Google Cloud Platform project, you'll want to delete the cluster to avoid incurring charges.
 
 1. Navigate to the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes view</walkthrough-editor-spotlight> in the Activity bar.
 
-2. Under the <walkthrough-editor-spotlight spotlightId="cloud-code-gke-explorer">Google Kubernetes Engine Explorer tab</walkthrough-editor-spotlight>, right-click on your cluster and select **Delete Cluster**.
+2. Under the <walkthrough-editor-spotlight spotlightId="cloud-code-gke-explorer">Clusters Explorer tab</walkthrough-editor-spotlight>, right-click on your cluster and select **Delete Cluster**.
```

Comment:
In some bits of the tutorial it's "**clusters** explorer tab" and here it's "**clusters explorer tab**" (spotlights in bold). Let's make sure what text is being spotlighted (and capitalization / etc) is consistent throughout the tutorial

Reply:
Fixed! :)

I also changed "**Cloud Code - Kubernetes sidebar**" to "**Cloud Code - Kubernetes** sidebar" (I double checked other tutorials to see which one is used so that this tutorial can be consistent)

Reply:
Thank you!!